src/form/form-selector.ts

```diff
@@ -73,12 +73,12 @@ export function createFormSelectorFactory(): FormSelectorFactory {
     }
 
     function processProvince(field: FormField, country?: Country): FormField {
-        const { subdivisions = [] } = country || {};
+        const { subdivisions = [], requiresState = false } = country || {};
```

Comment:
might be safer to not default to `false`, you might want to check if it's undefined and default to the existing behaviour if this property does not exist.

Reply:
removed default and added check to use default if requiresState is null or undefined.